content/en/blog/introduction_to_wamr_running_modes/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Introduction to WAMR running modes"
+description: ""
+excerpt: ""
+date: 2023-01-19T17:20:07+08:00
+lastmod: 2023-01-28T17:20:07+08:00
+draft: false
+weight: 50
+images: []
+categories: ['introduction']
+tags: []
+contributors: ['Tianlong Liang']
+pinned: false
+homepage: false
+---
+
+When you first meet WAMR, want to give it a taste, either using `iwasm` to run a hello world wasm application, or embed a WAMR into your hello world program, I bet you won't even sweat a bit. But when you're up to do something serious, maybe use it for real world applications and start to think about performance and response time, things changes. At this time, the different running modes (AOT, interpreter, fast interpreter, fast JIT, LLVM JIT, multi-tier JIT) we proudly made, aiming to provide user more adaptability and flexibility, could backfire and encumber user.
+
+Although we try to make CMake cache variables' name(CMake cache variables serve as the method to choose running mode at compile time) as explanatory as possible, because our background varies and the same term may mean differently to us, eventually, it can still cause confusion. And when it did, you may have to go back and forth from the documentation page to your develop environment, trying to understand each running mode are for, compile and test WAMR for your wasm app, see how does each running mode perform. We understand it's a chore (even worse, it maybe a dealbreaker for some people) to go through this process.
```

Comment:
`trying to understand each running mode are for` -> `trying to understand what each running mode is for`
`see how does each running mode perform` -> `see how each running mode performs`